src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+@import 'var';
+
+.card {
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  background: $background-white-color;
+  max-width: 198px;
+  margin-bottom: 500px !important;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    & img {
+      width: 100%;
+    }
+  }
+
+  &__title {
+    color: rgb(6, 11, 53);
+    font-size: $text-size;
+    font-weight: 500;
+    line-height: 17px;
+    padding-top: 40px;
+  }
+
+  &__code {
+    color: rgb(97, 96, 112);
```

Comment:
The color `rgb(97, 96, 112)` should be defined as an SCSS variable to ensure consistency and ease of updates.